app/views/merchants/dashboard.html.erb

```diff
@@ -11,10 +11,16 @@
 <section class="merchant_top_5_customers">
   <h3>Top Five Customers</h3>
   <% @merchant.top_five_customers.each do |customer| %>
-    <h3><%= customer.customer_name %> | Successful Transactions w/ Merchant: <%= customer.transaction_result %></h3>
+    <h3> <%= customer.customer_name %> | Successful Transactions w/ Merchant: <%= customer.transaction_result %> </h3>
   <% end %>
 </section>
 
 <section class="merchant_items_to_be_shipped">
   <h3> Items Ready to Ship </h3>
+    <% @merchant.rts_items.each do |invoice_item| %>
+      <h3> <%= invoice_item.item.name %> | 
+      <%= link_to "Invoice #{invoice_item.invoice_id}", "/merchants/#{@merchant.id}/invoices/#{invoice_item.id}", method: :get %> |
+      <%= invoice_item.invoice.created_at.strftime("%A, %B %e, %Y - %H:%M:%S") %>
```

Comment:
We might have to refactor this because I think it violates a certain rule but not sure - it works though!